conda/cli/conda_argparse.py

```diff
@@ -1708,6 +1708,13 @@ def add_parser_solver_mode(p):
         default=NULL,
         help="Ignore pinned file.",
     )
+    solver_mode_options.add_argument(
+        "--prerelease-behavior",
```

Comment:
I think this can just be `--prerelease`. Behavior sounds redundant here, in the context of a choice flag. Let's hear from other reviewers though, I am not a native speaker :)

Reply:
I thought it would be clearest for this to exactly match the config parameter, especially since there is another config parameter `prerelease_allowed_packages`. The current conda argparse configuration already allows abbreviations, so users will actually be able to write `--prerelease limit` (although I have not tried this myself).

In practice, I expect that this option will not be used that often. Most organizations will want to just tell everyone to configure the option in their global or project `.condarc`.